index.bs

```diff
@@ -3962,6 +3978,10 @@ When this method is invoked, the [=authenticator=] MUST perform the following pr
 1. If any error occurred while generating the [=assertion signature=], return an error code equivalent to "{{UnknownError}}" and
     terminate the operation.
 
+1. Let |authenticatorAttachment| be <code>platform</code> or <code>cross-platform</code> based on the transport returned by authenticator following successful authentication. 
```

Comment:
It's the User Agent's responsibility to tell the attachment, not the authenticator's. Remove this section.

Reply:
I agree this is the user agents responsibility, but I think parts of this could be merged into the section discussed in https://github.com/w3c/webauthn/pull/1668/files#r712433782 (which would in turn be moved to line 2256 as noted there). The merged text should also use the `{{AuthenticatorAttachment/platform}}` and `{{AuthenticatorAttachment/cross-platform}}` references.